packages/admin/cms-admin/src/blocks/rte/extension/CmsLink/createCmsLinkToolbarButton.tsx

```diff
@@ -131,22 +131,48 @@ export function createCmsLinkToolbarButton({ link: LinkBlock }: CreateCmsLinkToo
                 <DialogContent>
                     <LinkBlock.AdminComponent state={newLinkState} updateState={setNewLinkState} />
                 </DialogContent>
-                <DialogActions>
-                    {linkEntity && (
-                        <Button onClick={handleRemove} color="primary">
-                            <FormattedMessage id="comet.rteExtensions.cmsLink.removeLink" defaultMessage="Delete Link" />
-                        </Button>
-                    )}
-                    <Button onClick={handleUpdate} color="primary">
-                        <FormattedMessage {...messages.ok} />
-                    </Button>
-                    <Button onClick={handleClose} color="primary">
+                <StyledDialogActions>
+                    <CancelButton onClick={handleClose} color="primary">
                         <FormattedMessage {...messages.cancel} />
-                    </Button>
-                </DialogActions>
+                    </CancelButton>
+                    <ButtonContainer>
+                        {linkEntity && (
+                            <DeleteButton onClick={handleRemove} color="primary">
+                                <FormattedMessage id="comet.rteExtensions.cmsLink.removeLink" defaultMessage="Delete Link" />
+                            </DeleteButton>
+                        )}
+                        <OkayButton onClick={handleUpdate} color="primary">
+                            <FormattedMessage {...messages.ok} />
+                        </OkayButton>
+                    </ButtonContainer>
+                </StyledDialogActions>
             </Dialog>
         );
     }
 
     return ToolbarButton;
 }
+
+const StyledDialogActions = styled(DialogActions)`
+    display: flex;
+    flex-direction: column-reverse;
+    align-items: stretch;
+    gap: 10px;
+
+    ${({ theme }) => theme.breakpoints.up("md")} {
+        flex-direction: row;
+        justify-content: space-between;
+        align-items: center;
+        gap: 0;
+    }
+`;
+
+const ButtonContainer = styled("div")`
+    display: contents;
+
+    ${({ theme }) => theme.breakpoints.up("md")} {
+        display: flex;
+        flex-direction: row;
+        gap: 10px;
+    }
+`;
```

Comment:
@jamesricky is this the recommended way to position actions in a dialog? Could this be done with [Grid](https://mui.com/material-ui/react-grid2/)?

Reply:
For this one instance, yes. For all buttons in dialog in general, no. We shouldn't need that many custom styling in the application.

Reply:
We'll have a generic solution in the future for this: https://vivid-planet.atlassian.net/browse/COM-1655